### ISSUE TITLE
Google free version

### ISSUE THREAD

**Issue by mueller-ma** [ostera/tldr.jsx#83]
Hi,

I want to host tldr.jsx on a local server. What is the easiest way to get a version no third partie tools (e.g. google analytics)

Maybe include such an option in the makefile.

**Comment by ostera** [comment]
Good idea! I have a branch where I'm moving away from `envify` and just using gnu's `envsubst` to replace things like NODE_ENV, and other API tokens across the source code.

I'll add a check for environment to the script that includes the analytics, so it looks like:

```
if ( ${NODE_ENV} === "production" ) {
  // setup analytics here
}
```

So that renders to a `if ("local"==="production") { // bootstrap analytics here }` and thus never runs by default, but can still be turned on (by passing `NODE_ENV=production` to make) for debugging purposes.
